monitoring/daily_report.py

```
@@ -0,0 +1,654 @@
+            'legacy': """SELECT
+                (SELECT COUNT(*) FROM newsletter_emails WHERE subscribed = true) +
+                (SELECT COUNT(*) FROM newsletter_subscribers WHERE unsubscribed_at IS NULL)
+            """,
+        }
+
+        env = os.environ.copy()
+        env['PGPASSWORD'] = password
+
+        for key, query in queries.items():
+            try:
+                result = subprocess.run(
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)